### ISSUE TITLE
Bug - Landing gear does not extend after turning off engine master switches

### ISSUE THREAD

**Issue by linguisticmind** [legoboyvdlp/A320-family#322]
**Describe the bug**
Landing gear does not extend if power to the engines is killed, but the aircraft pitches down as if from the drag created by the extending gear. (I've been turning off master switches in flight to simulate an engine failure.)

**To Reproduce**
Video demonstration: https://youtu.be/uymG1CogP3c
Steps to reproduce the behaviour:
1. While in the air, turn off the engines' master switches
2. Extend the landing gear

**Expected behaviour**
Landing gear extending like in the current FGAddon version.

**System (please complete the following information):**
 - OS: Debian Stable (12, Bookworm)
 - FlightGear version: 2020.3.19
 - A320 Version: dev, 8869ad8

---

I've also noticed that the gravity gear extension handle says "PULL & TRIM" instead of "PULL & TURN". The FGAddon version says "PULL & TURN".

Current:

![fgfs-20240525021628](https://github.com/legoboyvdlp/A320-family/assets/54222750/7909b82f-541c-4a53-84e3-89a9979fc398)

FGAddon:

![fgfs-20240525021412](https://github.com/legoboyvdlp/A320-family/assets/54222750/8035b56a-c123-4ac9-b1f6-7fa6089d9b0e)

**Comment by L1011H** [comment]
> I've also noticed that the gravity gear extension handle says "PULL & TRIM" instead of "PULL & TURN". The FGAddon version says "PULL & TURN"

@linguisticmind This is now fixed, will be available after commit shortly, thanks.

**Comment by linguisticmind** [comment]
I recorded a video to demonstrate the issue more clearly: https://youtu.be/uymG1CogP3c

@Octal450 If it's not too much trouble, I was wondering if you could take a look. I don't know who gets notifications about comments to this issue. I'm trying to practice engine failures, and there doesn't seem to be an option for that in the 'System Failures' menu. So I've been turning the master switches off in flight to simulate that, and came across this bug. Maybe there is a way to practice engine failures without touching the master switches that I don't know about? Could an engine failure option in the 'System Failures' menu be a feature request?

Also just wanted to say thanks a lot to everyone who is involved in this awesome project for their great work!

**Comment by Octal450** [comment]
Yeah, I'm not getting notif's since it's not my repo. Best to assign me.

I'll check this out.

Regards,
Josh

**Comment by Octal450** [comment]
It pitches down due to loss of C* law in DIRECT LAW.

Gear doesn't extend because HYD isn't up and alt gear lever doesn't animate yet.

Kind Regards,
Josh

**Comment by linguisticmind** [comment]
I see. Thanks for explaining.

I didn't realize that hydraulics wouldn't be able to extend the gear under those conditions. In case someone else is curious, I did some googling, and [this answer](https://planes.quora.com/Why-does-A320-emergency-procedure-Dual-hydraulic-Loss-B-Y-say-Gear-Gravity-extension-though-the-landing-gear-is-powe) explains it pretty well.

Hope the gravity extension gets implemented. Is there perhaps some workaround that can be used to extend the gear under those circumstances in the meantime?

**Comment by linguisticmind** [comment]
Never mind. Got it.

```xml
  <!-- Bind Shift-Alt-F9 to gravity gear extension. -->
  <!-- Note: The normal landing gear lever still has to be in the down position. -->
  <key n='265'>
    <name>F9</name>
    <desc>Gravity gear extension</desc>
    <mod-shift>
      <mod-alt>
        <binding>
          <command>property-toggle</command>
          <property>/controls/hydraulic/switches/gear-gravity-ext</property>
        </binding>
      </mod-alt>
    </mod-shift>
  </key>
```
